tests/compile-fail/derive_for_enum_fails.rs

```diff
@@ -0,0 +1,10 @@
+extern crate rocket;
```

Comment:
`extern crate` should not be used in Rust 2018.

Reply:
In this situation importing from the crate prelude does not work.